src/main/java/it/gov/pagopa/arc/config/OAuth2LoginConfig.java

```diff
@@ -28,7 +27,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
             )
             .redirectionEndpoint(redirection ->
                 redirection
-                    .baseUri("/token/*")
```

Comment:
i think `oneidentity` is appended by default.` /token` should be enough. @oleksiybozhykntt what do you think?

Reply:
i have checked the default configuration and the value is "/login/oauth2/code/*" the star is required to indicate to match any redirect uri that starts with /token/* , otherwise we must know what should be the full url but it is impossible because "code" and "state" changes continuosly

Reply:
that field should indicate only the baseUri excluded code and state. That's why the star should not be needed.